### DISCUSSION TITLE
Why jvm metrics don't include daemon threads？

### DISCUSSION THREAD

**Discussion by tuhao1020** [open-telemetry/opentelemetry-specification#2769] [Q&A]
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics

`ThreadMXBean` already provides the `getDaemonThreadCount` method

**Comment by arminru** [comment]
@tuhao1020 They are included, actually:

There is a metric called `process.runtime.jvm.threads.count` for which a boolean attribute `daemon` is defined. 
On your metrics backend, the metric will report the total count of all threads and by filtering on or splitting by `daemon` you can distinguish them.

See https://github.com/open-telemetry/opentelemetry-specification/blob/v1.16.0/specification/metrics/semantic_conventions/runtime-environment-metrics.md?plain=1#L76
(rightmost columns in the table).
